File: DeskHubSharpRevised/Models/Email.cs
namespace DeskHubSharpRevised.Models;

public class Email
{
    private string _toEmail = "wjmiller2016@gmail.com";
    private string _fromEmail = "wjmiller2016@gmail.com";
    private string _passwordEmail = "password";

    public string Password
    {
        get { return _passwordEmail; }
        set { _passwordEmail = value; }
    }

    public string FromEmail
    {
        get { return _fromEmail; }
        set { _fromEmail = value; }
    }

    public string ToEmail
    {
        get { return _toEmail; }
        set { _toEmail = value; }
    }

    /// <summary>
    /// Constructor for the Email class
    /// </summary>
    public Email()
    {

    }
}

File: ModelsLib/Email.fs
namespace DeskHubSharpRevised.Models

type Email() =
    let mutable _toEmail: string = "wjmiller2016@gmail.com"
    let mutable _fromEmail: string = "wjmiller2016@gmail.com"
    let mutable _password: string = "password"
    
    member this.ToEmail
        with get() = _toEmail
        and set(value: string) = _toEmail <- value
    member this.FromEmail
        with get() = _fromEmail
        and set(value: string) = _fromEmail <- value
    member this.Password
        with get() = _password
        and set(value: string) = _password <- value